src/app/api/core/types/api.ts

```diff
@@ -0,0 +1,10 @@
+export enum Resource {
+  Tasks = 'Task',
+  WorkflowState = 'WorkflowStates',
```

Comment:
what is workflow state? also lets follow one convention, may be singular is better

Reply:
@sazanrjb I noticed this and fixed this in PR #10. Cherrypicking that change to this PR too.